data.sql

```diff
@@ -7,3 +7,39 @@ INSERT INTO ANIMALS (NAME,DATE_OF_BIRTH,WEIGHT_KG,NEUTERED,ESCAPE_ATTEMPTS)
  ('Devimon','05-12-2021',11,true,5);
 
 
+-- Day 2
+INSERT INTO ANIMALS (NAME,DATE_OF_BIRTH,WEIGHT_KG,NEUTERED,ESCAPE_ATTEMPTS)
+ VALUES ('Charmander','02-08-2020',-11,false,0),
+ ('Plantmon','11-15-2021',-5.7,true,2),
+ ('Squirtle','04-02-1993',-12.13,false,3),
+ ('Angemon','06-12-2005',-45,true,1),('Boarmon','06-07-2005',20.4,true,7),
+ ('Blossom','10-13-1998',17,true,3),('Ditto','05-14-2022',22,true,4);
+
+
+BEGIN TRANSACTION;
+UPDATE ANIMALS 
+SET SPECIES = NULL;
+ROLLBACK ;
+
+
+BEGIN TRANSACTION; 
+UPDATE ANIMALS SET SPECIES = 'digimon' WHERE NAME LIKE '%mon';
+UPDATE ANIMALS SET SPECIES = 'pokemon' WHERE SPECIES IS NULL;
+COMMIT TRANSACTION ;
+
+BEGIN TRANSACTION ;
+DELETE FROM ANIMALS;
+ROLLBACK ;
+
```

Comment:
- [OPTIONAL] Just like I suggested above, it will be once to verify here after the rollback.

See example:
``` sql
SELECT COUNT(*) FROM ANIMALS;
```